src/HVX/Internal/DCP.hs

```diff
@@ -21,77 +21,77 @@ class GetVex (v :: Vex) where
 
 -- | convexity
 data Vex = Affine | Convex | Concave | Nonvex
-instance GetVex Affine  where getVex _ = "Affine"
-instance GetVex Convex  where getVex _ = "Convex"
-instance GetVex Concave where getVex _ = "Concave"
-instance GetVex Nonvex  where getVex _ = "Nonvex"
+instance GetVex 'Affine  where getVex _ = "Affine"
+instance GetVex 'Convex  where getVex _ = "Convex"
+instance GetVex 'Concave where getVex _ = "Concave"
+instance GetVex 'Nonvex  where getVex _ = "Nonvex"
 
 class ValidVex (v :: Vex) where
     validVex :: e (v :: Vex) (m :: Mon) -> e v m
     validVex = id
 
-instance ValidVex Affine
-instance ValidVex Convex
-instance ValidVex Concave
+instance ValidVex 'Affine
+instance ValidVex 'Convex
+instance ValidVex 'Concave
 
 class GetMon (m :: Mon) where
   getMon :: e (v :: Vex) (m :: Mon) -> String
 
 -- | monotonicity
 data Mon = Const | Nondec | Noninc | Nonmon
-instance GetMon Const  where getMon _ = "Const"
-instance GetMon Nondec where getMon _ = "Nondec"
-instance GetMon Noninc where getMon _ = "Noninc"
-instance GetMon Nonmon where getMon _ = "Nonmon"
+instance GetMon 'Const  where getMon _ = "Const"
+instance GetMon 'Nondec where getMon _ = "Nondec"
+instance GetMon 'Noninc where getMon _ = "Noninc"
+instance GetMon 'Nonmon where getMon _ = "Nonmon"
 
 -- | invert convexities
 type family FlipVex v where
-  FlipVex Convex  = Concave
-  FlipVex Concave = Convex
-  FlipVex Affine  = Affine
-  FlipVex Nonvex  = Nonvex
+  FlipVex 'Convex  = 'Concave
+  FlipVex 'Concave = 'Convex
+  FlipVex 'Affine  = 'Affine
+  FlipVex 'Nonvex  = 'Nonvex
 
 -- | invert monotonicities
 type family FlipMon m where
-  FlipMon Const  = Const
-  FlipMon Nondec = Noninc
-  FlipMon Noninc = Nondec
-  FlipMon Nonmon = Nonmon
+  FlipMon 'Const  = 'Const
+  FlipMon 'Nondec = 'Noninc
+  FlipMon 'Noninc = 'Nondec
+  FlipMon 'Nonmon = 'Nonmon
 
 -- determines the convexity of a function applied to an expression
 -- "newexpr = apply f expr"
 type family ApplyVex vf mf ve me where
-  ApplyVex vf      Const  ve      me    = Affine
-  ApplyVex vf      mf     ve      Const = Affine
-  ApplyVex vf      mf     Affine  me    = vf
-  ApplyVex Affine  Nondec ve      me    = ve
-  ApplyVex Affine  Noninc ve      me    = FlipVex ve
-  ApplyVex v       Nondec v       me    = v
-  ApplyVex Convex  Noninc Concave me    = Convex
-  ApplyVex Concave Noninc Convex  me    = Concave
-  ApplyVex vf      mf     ve      me    = Nonvex
+  ApplyVex vf      'Const   ve       me     = 'Affine
```

Comment:
...? I haven't changed anything, this is your PR. I was just suggesting that the `'Const` should line up with the things below it.